.storybook/main.js

```diff
@@ -44,6 +44,15 @@ module.exports = {
       ],
     });
 
+    config.resolve = {
+      ...config.resolve,
+      alias: {
+        ...config.resolve.alias,
+        '@emotion/core': '@emotion/react',
+        'emotion-theming': '@emotion/react',
+      },
+    };
```

Comment:
Storybook..... what more do i need to say. 
We had to add this because when we upgraded to emotion 11, our theming was broken. Technically our code was not broken, but since we're on version 6.3 of storybook, storybook was using `emotion 10` for theming which was breaking our themed stories. This is a workaround to tell Storybook to use emotion 11 packages.

Here's the issue outlined with the solution posted in the comments: https://github.com/storybookjs/storybook/issues/13145